code/src/WijmoProvider/Features/ValidationMark.ts

```diff
@@ -399,6 +400,11 @@ namespace WijmoProvider.Feature {
             );
         }
 
+        public setRowStatus(rowNumber: number, isValid: boolean): void {
+            // set invalidRows with row number and flag that checks if status isValid and if there are invalid values on metadata
+            this._setInvalidRows(rowNumber, isValid);
```

Comment:
the method this._setInvalidRows should be renamed as well 🤔

Reply:
fixed